src/client/Microsoft.Identity.Client/Platforms/Android/AndroidLegacyCachePersistance.cs

```diff
@@ -9,7 +9,11 @@
 
 namespace Microsoft.Identity.Client.Platforms.Android
 {
+#if MAUI
```

Comment:
Where is MAUI defined?

Reply:
Built in ?

Reply:
It's not built in. You've defined it in the csproj at line 158.